File: 프로그래머스/1/86491. 최소직사각형/최소직사각형.py
def solution(sizes):
    answer = 0
    ga = 0
    se = 0
    for i in sizes :
        i.sort()
        print(i[1])
        if i[1] > ga :
            ga = i[1]
            
    for i in sizes :
        if i[0] > se :
            se = i[0]
    return ga*se

File: 프로그래머스/1/86491. 최소직사각형/최소직사각형.java
class Solution {
    public int solution(int[][] sizes) {
        int answer = 0;
        int ga_max = 0;
        int se_max = 0;
        
        for(int i=0; i<sizes.length; i++){
            if (sizes[i][0] < sizes[i][1]){
                int change = sizes[i][0];
                sizes[i][0] = sizes[i][1];
                sizes[i][1] = change;
            } else{
            }
            
            if(sizes[i][0] > ga_max){
                ga_max = sizes[i][0];
            }
            if(sizes[i][1] > se_max){
                se_max = sizes[i][1];
            }
        }
        
        
        
        answer = ga_max * se_max;
        
        return answer;
    }
}